src/main/java/bio/terra/filesystem/FireStoreDependencyDao.java

```diff
@@ -143,7 +142,7 @@ public void storeSnapshotFileDependency(Dataset dataset, String snapshotId, Stri
                     // no dependency object yet. Let's make one
                     FireStoreDependency fireStoreDependency = new FireStoreDependency()
                         .snapshotId(snapshotId)
-                        .datasetId(dataset.getId().toString())
+                        .datasetId(dataset.getId().toString()) // TODO: does this still need to be here?
```

Comment:
No. You can remove that field from the FireStoreDependency class and (obviously) not populate it here.